build.gradle

```diff
@@ -21,7 +21,7 @@ idea.project.settings {
 
 import net.ltgt.gradle.errorprone.CheckSeverity
 
-['jib-core', 'jib-gradle-plugin-extension-api', 'jib-maven-plugin-extension-api'].each { projectName ->
+['jib-core', 'jib-gradle-plugin', 'jib-gradle-plugin-extension-api', 'jib-maven-plugin-extension-api'].each { projectName ->
```

Comment:
Why do we want to apply `java-library` for `jib-gradle-plugin`? It's not a Java library. And it's not in parity with `jib-maven-plugin` either.

Reply:
ah, my comment is missing when I rewrote the PR. The java-gradle-plugin now auto applies the java-library plugin, so we must do this, I'll add in the comment.

Reply:
If `java-gradle-plugin` auto-applies `java-library`, then doesn't it mean we don't have to apply it manually here? (That is, no need to include `jib-gradle-plugin` in this line?

And the Travis "pr" build failed and this seems related. Interestingly, however, the "push" build didn't fail.

Reply:
Right the reason is we need to apply `Java-library` before `java`

Reply:
or we get some error where the something is already configured that the `java-library` plugin wants to modify.